topo/node/cisco/cisco_test.go

```diff
@@ -656,3 +637,112 @@ func TestNew(t *testing.T) {
 		})
 	}
 }
+
+func TestNodeStatus(t *testing.T) {
+
+	ki := fake.NewSimpleClientset(&corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "pod1",
+		},
+		Status: corev1.PodStatus{
+			Phase: corev1.PodRunning,
+		},
+	})
+	ki.PrependReactor("get", "pods", func(action ktest.Action) (handled bool, ret runtime.Object, err error) {
+		if action.GetSubresource() == "log" {
```

Comment:
you can setup the clientset inside of each test case to directly handle the log calls as you would expect

Reply:
Looks like logs are not easily supported through the fake client, found the logs are hardcoded to be `fake log` after users had issues with this: https://github.com/kubernetes/kubernetes/pull/91485

So for a fix I just have test cases change the regexp to simulate gettting logs we want vs undesirerable logs